WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 299-190-182
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 299-190-182
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122  * Detected change in 'D:\\pyCode\\flaskProject\\Flask_home\\home\\api\\users.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 299-190-182
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
ERROR users.py:10 err ssss
INFO _internal.py:122 127.0.0.1 - - [25/Jul/2020 14:50:27] "GET /api/v1.0/index HTTP/1.1" 200 -
INFO _internal.py:122  * Detected change in 'D:\\pyCode\\flaskProject\\Flask_home\\home\\api\\users.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 299-190-182
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
